packages/zowe-explorer/src/job/actions.ts

```diff
@@ -285,12 +285,36 @@ export async function deleteCommand(job: IZoweJobTreeNode, jobsProvider: IZoweTr
         return;
     }
 
-    // delete selected nodes
-    if (nodes.length > 0) {
-        for (const node of nodes) {
-            await jobsProvider.delete(node);
-            deletedNodes.push(`${node.job.jobname}(${node.job.jobid})`);
-        }
+    // delete selected multiple nodes
+    if (nodes.length > 1) {
+        await vscode.window.withProgress(
+            {
+                location: vscode.ProgressLocation.Notification,
+                title: localize("deleteJobPrompt.deleteCounter", "Deleting nodes"),
+                cancellable: true,
+            },
+            async (progress, token) => {
+                const total = 100;
```

Comment:
What does this mean? And why is it 100? Can we also move this to globals.ts?

Reply:
because, it is measured in percentage so, 100 is the base, for it : )

Reply:
ok. let's leave this as is then